python-sdk/tutorials/automl-with-azureml/image-classification-multilabel/auto-ml-image-classification-multilabel.ipynb

```diff
@@ -636,35 +684,127 @@
     "Finally, let's test our deployed web service to predict new images. You can pass in any image. In this case, we'll use a random image from the dataset and pass it to the scoring URI."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "from io import BytesIO\n",
+    "from PIL import Image\n",
+    "\n",
+    "\n",
+    "def img_to_base64(img_path):\n",
+    "    \"\"\"Load and convert pillow image to base64-encoded image\"\"\"\n",
+    "\n",
+    "    img = Image.open(img_path)\n",
+    "    imgio = BytesIO()\n",
+    "    img.save(imgio, img.format)\n",
+    "    img_str = base64.b64encode(imgio.getvalue())\n",
+    "    return img_str.decode('utf-8')\n",
+    "\n",
+    "\n",
+    "def base64_to_img(base64ImgString):\n",
+    "    base64Img = base64ImgString.encode('utf-8')\n",
```

Comment:
super nit: maybe we could use either camel or snake case in notebook. maybe base64_image, base64_img_str, etc